Gemfile

```diff
@@ -3,5 +3,4 @@ source 'https://rubygems.org'
 # Specify your gem's dependencies in amazon_ssa_support.gemspec
 gemspec
 
-gem "handsoap", "=0.2.5.5", :require => false, :source => "http://rubygems.manageiq.org"
```

Comment:
This is needed because this pulls in manageiq-smartstate and _that_ needs the custom gem.